File: mobilCihazdanSunucuUzerndekVertabannaKaytEkleme/app/src/main/java/com/example/mobilcihazdansunucuuzerndekvertabannakaytekleme/MainActivity.java
package com.example.mobilcihazdansunucuuzerndekvertabannakaytekleme;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

import Models.Deneme;
import RestApi.ManagerAll;
import retrofit2.Call;
import retrofit2.Callback;
import retrofit2.Response;

public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        istek();

    }


    public void istek() {
        Call<Deneme> abv = ManagerAll.getInstance().ekle();
        abv.enqueue(new Callback<Deneme>() {
            @Override
            public void onResponse(Call<Deneme> call, Response<Deneme> response) {
                Toast.makeText(getApplicationContext(), response.body().toString(), Toast.LENGTH_SHORT).show();
            }

            @Override
            public void onFailure(Call<Deneme> call, Throwable t) {
                Toast.makeText(getApplicationContext(), "basarisiz", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: KotlinDer18InterstitialAds/app/src/main/java/com/example/kotlinder18interstitialads/MainActivity.kt
package com.example.kotlinder18interstitialads

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import com.google.android.gms.ads.AdListener
import com.google.android.gms.ads.AdRequest
import com.google.android.gms.ads.InterstitialAd
import kotlinx.android.synthetic.main.activity_main.*

/**
 * bu reklamımım idsi Interstitial: ca-app-pub-4718213835461634/9354875604
 google play'e yüklerken lazım oluyor bu
 * Interstitial test id: ca-app-pub-3940256099942544/1033173712
 */
class MainActivity : AppCompatActivity() {

    private lateinit var mInterstitialAd: InterstitialAd

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mInterstitialAd = InterstitialAd(this)
        mInterstitialAd.adUnitId = "ca-app-pub-3940256099942544/1033173712"  //bu id deneme id'si dikkat et
        mInterstitialAd.loadAd(AdRequest.Builder().build())

        //ben geri gelsem de her seferinde 2.aktiviteye giderken reklam açılsın istiyorsam eğer
        mInterstitialAd.adListener = object : AdListener() {
            override fun onAdClosed() {
                mInterstitialAd.loadAd(AdRequest.Builder().build())
            }
        }

        basGit.setOnClickListener {
            val intent=Intent(applicationContext,Main2Activity::class.java)
            startActivity(intent)
            //bu tek seferliğe mahsus çalışıyor.
            if (mInterstitialAd.isLoaded) {
                mInterstitialAd.show()
            }
        }
    }
}
